.changeset/giant-beers-grow.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
I'm going to try this again.  I think there was an issue with me creating an "empty" file after first creating a non-empty file. 🤷‍♂️